app/controllers/home_controller.rb

```diff
@@ -2,5 +2,6 @@ class HomeController < ApplicationController
   after_action :verify_authorized, except: [:index]
 
   def index
+    redirect_to current_user.after_sign_in_path if user_signed_in?
```

Comment:
this seems not necessary. devise redirects to the home controller for some reason, but then you redirect again? nope, lets handle this at the devise level. there is likely a hook in the devise omniauth controller for doing just this, similar to the after_sign_in_path method in the sessions controller.